validator/slashing-protection/cli_import.go

```diff
@@ -0,0 +1,46 @@
+package slashingprotection
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/shared/fileutil"
+	"github.com/prysmaticlabs/prysm/validator/accounts/prompt"
+	"github.com/prysmaticlabs/prysm/validator/db"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	slashingProtectionFormat "github.com/prysmaticlabs/prysm/validator/slashing-protection/local/standard-protection-format"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+// ImportSlashingProtectionCLI is the CLI command for importing slashing protection into a JSON.
+func ImportSlashingProtectionCLI(cliCtx *cli.Context, valDB db.Database) error {
+	if valDB == nil {
```

Comment:
This won't work due to https://mangatmodi.medium.com/go-check-nil-interface-the-right-way-d142776edef1. We should perhaps due a type assertion here and check if it is a *kv.Store instead